config.go

```diff
@@ -43,7 +43,7 @@ const (
 	defaultBlockMaxSize      = 375000
 	blockMaxSizeMin          = 1000
 	blockMaxSizeMax          = wire.MaxBlockPayload - 1000
-	defaultBlockPrioritySize = 50000
+	defaultBlockPrioritySize = 0
```

Comment:
I wouldn't change the default high priority area.  That is for **non-stake** transactions and will be more important later when there is contention for space.

Stake transactions should *always* be higher priority than regular transactions, so this should have no effect whatsoever on them.